src/lib.rs

```diff
@@ -5,53 +5,24 @@ use std::fs::File;
 use std::io::prelude::*;
 use std::error::Error;
 
-#[derive(FromPyObject)]
-pub struct ModuleSpec {
-    name: String,
-    origin: String
-}
+// TODO: Do tests actually need this?
+pub mod minimal_parser;
+pub use minimal_parser::*;
```

Comment:
Hmmm I haven't seen this sort of thing before, maybe be able to get rid of the use unless things in `minimal_parser` aren't declared `pub` ? 

Note - you can do `pub(crate)` if you want things to be public within this crate like for testing, but not public to external crates